### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by daslerr**
Updates readme to include more basic background information pulled from other CYF docs.

## What does this change?

This PR updates the readme to (hopefully) give a bit of background about what's going on. I split the available information into 'About our curriculum' and 'About the platform' and pulled in some explanatory info from other docs and pages across the CYF ecosystem. 

### Common Content?

No, this does not add any info to the common-content module. 

### Common Theme?

No, this does not add any features or fix any bugs for the common-theme module

### Org Content?

This PR doesn't change or block anyone else's content (so far as I know). I only changed the readme.

### CODE REVIEW

**Comment by daslerr** [comment]
@SallyMcGrath Please check to make sure I've characterized the CYF approach appropriately. Thanks!

File: README.md [hunk 1]
```diff
@@ -1,19 +1,34 @@
...
-## What is it?
+## What is this?
 
-This is a white label multi-tenant platform for us and our code school friends to use to manage our curricula. It's built on top of [Hugo](https://gohugo.io/) and [Netlify CMS](https://www.netlifycms.org/).
+This repo houses both our main curriculum content and our curriculum platform. 
 
-The platform, layout, styles and components are all developed in the Hugo module [/common-theme](/common-theme).
+## About our curriculum
+
+Code Your Future is a volunteer-led community of tech professionals, and we teach people to code for free. We consider ourselves to be professionals helping other professionals, so our insructional approach is not hierarchical, and we encourage our volunteers to test out new material early and often. Our main curriculum (the repo you're looking at now) threads a coherent line through all of this activity, so whatever worlds we explore, everyone can come back here to find out what to do next.
```
Comment (illicitonion) [inline README.md:13]:
```suggestion
Code Your Future is a volunteer-led community of tech professionals, and we teach people to code for free. We consider ourselves to be professionals helping other professionals, so our instructional approach is not hierarchical, and we encourage our volunteers to test out new material early and often. Our main curriculum (the repo you're looking at now) threads a coherent line through all of this activity, so whatever worlds we explore, everyone can come back here to find out what to do next.  
```

**Review by illicitonion — COMMENTED** [review]
This looks great, really useful detail and context for folks, thanks so much!

I left one non-trivial comment, happy to hop on a quick call to discuss if you'd like :)

File: README.md [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
+## About the curriculum platform
+
+Our self-developed curriculum platform is called Common. It is a white label multi-tenant platform for us and our code school friends to use to manage our curricula. It's built on top of [Hugo](https://gohugo.io/) and [Netlify CMS](https://www.netlifycms.org/).
+
+The platform, layout, styles and components are all developed in the Hugo module [/common-theme](/common-theme).
 
 Multi-language support is provided by [Hugo's i18n support](https://gohugo.io/content-management/multilingual/).
 
-Each org or team configures its own Hugo site that mounts the common theme and content modules, and then makes any customisations they need and deploys it wherever they want. You can use any repo to do this and your site can be as simple as a `hugo.toml` and a `content/_index.md`.
+Common is tightly integrated with GitHub. Really the main content should always be project work in GitHub repositories, and any site generated with Common should be a viewer for that content.
```
[the remark this answers is not in the document]
Comment (daslerr) [reply README.md:29]: I actually pulled that line directly from the Common docs section on repos. here: https://common.codeyourfuture.io/more-sources/repos/ . I took it to mean that whatever you're using Common for, the meat of it should be in GitHub and the site on top is just for making it appear more approachable.

[the remark this answers is not in the document]
Comment (illicitonion) [reply README.md:31]: Ha! Let's leave it as this PR has it written for now :) Thanks!

**Review by illicitonion — APPROVED** [review]

**Pull request merged**